app/styles/base.css

```diff
@@ -491,6 +491,13 @@ html {
     margin: 5px;
 }
 
+.noVNC_panel label > button,
+.noVNC_panel label > select,
+.noVNC_panel label > textarea,
+.noVNC_panel label > input:not([type=checkbox]):not([type=radio]) {
```

Comment:
Did you consider doing a cleanup to use `:is()`?

Reply:
No, not right now. Would be nice though